static/app/views/dashboardsV2/widgetCard.tsx

```diff
@@ -185,6 +195,20 @@ class WidgetCard extends React.Component<Props> {
       }
     }
 
+    if (organization.features.includes('dashboards-edit')) {
+      menuOptions.push(
+        <StyledMenuItem
+          key="duplicate-widget"
+          data-test-id="duplicate-widget"
+          name="Duplicate Widget"
```

Comment:
Do we need this? I dug around and it looks like `StyledMenuItem` renders a `li` and I don't think `name` is necessary?

Reply:
yeah this one isn't needed, leftover from writing tests

Reply:
2f325784a7b17d290de713c4406bcf7699f4dc05